projects/burnr/src/components/PopoverInfo.tsx

```diff
@@ -43,13 +43,13 @@ const PopoverInfo: FunctionComponent<Props> = ({ children }: Props) => {
 
   return (
     <>
-      <a
+      <button
```

Comment:
is there a reason why <a> was converted to <button>?

Reply:
yes, accessibility. An `a` tag must have a valid `href` attribute, otherwise we should use a button. There was an eslint-rule that caught this. Thankfully as a warning, not an error, but since I noticed that we have custom styles I thought that it was better to make the change.